### ISSUE TITLE
Webcam act like a printer

### ISSUE THREAD

**Issue by Yokopoko9** [feelfreelinux/octo4a#195]
I tried many webcams on my android tv box Bqeel R2 Plus. All of these are correctly detected and fully functional with camera apps, but not with Octo4a. Octo4a seems to treat them like a printer. 

[Screenshoot](https://ibb.co/gPWcVd0)
[Screenshoot 2](https://ibb.co/sCk0YDY)
[Screenshoot 3](https://ibb.co/2WMMVF3)

**Comment by emailcraigs** [comment]
Did  you ever figure anything out on this? I have the same issue. My Logitech B910 shows up under the Printer is Connected area along with my printer, but as an Unknown serial driver. Seems that we have to inject a driver for the camera into the Octo4a program, but I have no idea how to do that. LOL. I'll post back here if I figure anything out.

**Comment by emailcraigs** [comment]
Funny thing.....I just tried to select the unknown serial driver and it flashed up with requesting permission but didnt check mark it. I went to the web UI and voila its is showing my camera. I clicked the Tap to show preview and it shows up there as well. I couldn't get this last night. The only thing I did is turn it all off and go to bed. Before that it crashed at each step...install printer...install camera, etc., but I'm serious when I say that each time it crashed and I rebuild it, things seem to work. My camera is a Logitech B910.

**Comment by Yokopoko9** [comment]
> Funny thing.....I just tried to select the unknown serial driver and it flashed up with requesting permission but didnt check mark it. I went to the web UI and voila its is showing my camera. I clicked the Tap to show preview and it shows up there as well. I couldn't get this last night. The only thing I did is turn it all off and go to bed. Before that it crashed at each step...install printer...install camera, etc., but I'm serious when I say that each time it crashed and I rebuild it, things seem to work. My camera is a Logitech B910.

I tried everything to replicate what you did! Nothing worked for me with Microsoft LifeCam HD-600 and Aukey PC-LM4 webcams.

**Comment by emailcraigs** [comment]
It seems to be hit or miss for sure. The only thing I can suggest is to clear the data and cache in android for octo4a, then run the install program again, let it install and see what it does. It seems like each time I did that something else started working. Honestly the more I have learned the more I'm not sure if Octoprint is worth messing with. There are serious problems with letting it do the printing. Its all tied up in one process so when something else grabs the CPU's attention.....it stops sending print data! Its crazy. That leads to blobs in your printed items as it pauses when the buffer runs out. But printing from the SD card limits most of what Octoprint is made for in the first place. Sucks.
